staging_docs/user/tutorials/00-overview.md

```diff
@@ -31,3 +31,4 @@ If you'd like to watch a recent talk about Pulp Container and see it in action,
 - Sign images and host Cosign signatures, SBOMS and attestations
 - Sign images and host Atomic signature type via extensions API
 - Host content via pull-through caching distributions
+- Replicate distributions from a specified Pulp instance, automatically creating the necessary remotes, repositories (those will always be created as read-only), and distributions to serve the same content
```

Comment:
- Discover and replicate distributions to serve the same content as the upstream Pulp